### PR TITLE
Add Account Bank Holidays test

### PR DESCRIPTION
**Pull request by huwd**
The GOV.UK Accounts team is running a discovery to explore
personalisation of GOV.UK. We are starting with a light touch, entirely
client side offer for users to remember their choice of bank holiday
nation.

We would like to introduce this gradually and so will stagger the number
of users who get offered this choice as we iterate and further test the
feature

### CODE REVIEW

File: configs/dictionaries/accountbankhols_percentages.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+A: 50
```
Comment (huwd) [inline configs/dictionaries/accountbankhols_percentages.yaml:2]: WIP PA to confirm

File: configs/dictionaries/accountbankhols_percentages.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+B: 50
```
Comment (huwd) [inline configs/dictionaries/accountbankhols_percentages.yaml:3]: WIP PA to confirm

File: configs/dictionaries/accountbankhols_percentages.yaml (hunk 1 above)
Comment (barrucadu) [reply configs/dictionaries/accountbankhols_percentages.yaml:2]: Maxwell has said that 50/50 is a good split.

Comment (barrucadu) [reply configs/dictionaries/accountbankhols_percentages.yaml]: And now Daisy has said a 20/80 split, which Maxwell has agreed to.

**Review by barrucadu — APPROVED** [review]

**Pull request merged**